include/lmms_math.h

```diff
@@ -2,6 +2,7 @@
  * lmms_math.h - defines math functions
  *
  * Copyright (c) 2004-2008 Tobias Doerffel <tobydox/at/users.sourceforge.net>
+ * Copyright (c) 2024 Roshan M R (AKA Ross Maxx) <mrroshan127/at/gmail/dot/com>
```

Comment:
It looks like this PR rearranges existing code, and deletes some obsolete stuff. Is there anything new here which is actually copyrightable?

Reply:
Ok I'll remove the copyright then.

Reply:
done